scheduling/pulse.py

```diff
@@ -0,0 +1,15 @@
+import os
+from pyspark import SparkConf, SparkContext
+from pyspark.sql import SQLContext
+from python_etl.testpilot import pulse
+
+if __name__ == "__main__":
+    conf = SparkConf().setAppName('python_etl')
+    sc = SparkContext(conf=conf)
+    sqlContext = SQLContext(sc)
+
+    # Make bdist_egg available to executer nodes
+    sc.addPyFile(os.path.join('dist', os.listdir('dist')[0]))
+
+    # Run job
+    pulse.etl_job(sc, sqlContext)
```

Comment:
The date is always 'yesterday' for this job (calculated [here](https://github.com/mozilla/python_etl/blob/master/python_etl/testpilot/utils.py#L7)).

For context, I want to get this version of the job checked in to work around [ATMO Issue 326](https://github.com/mozilla/telemetry-analysis-service/issues/326). We've needed to run this job manually for that last couple of weeks and it's impeding analysis for our clients. 

I'm not too concerned with date consistency for now since this job runs in <1h. Retries should still be done on the same day. I plan on implementing a more fully featured CLI app this afternoon, but it may take some more time to test and implement. Thanks for the click tip. I'll be sure to include that!